### PR TITLE
Improve graphs in mail report

### LINKED ISSUE #941: Email: improve the readability of graphs
* Increase font size
* Enlargen the history graph a bit
* Add a separate section that detail the legends
*

### PR DESCRIPTION
**Pull request by yasassri**
**Purpose**

Improve graphs in mail report. Resolves #941

image

**Security checks**
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: reporting/src/main/resources/templates/summarized_email_report.mustache [hunk 1]
```diff
@@ -155,13 +155,32 @@
...
+ <b>Test Failures:</b> Number of builds(build consist of one infra combination) failed
+                       due to test failures.
+ <b>Infrastructure Errors:</b> Number of builds failed due to infrastructure failures.
+                          (e.g: Product deployment failures etc.)
+ <b>Tests Passed: </b>Tests Passed infra combination.
+                        </pre>
+                    </td>
+                    <td>
+                        <pre>
+ Build failure summary for 7 days. All the builds that ran during 24H time rage
```
Comment (azinneera) [inline reporting/src/main/resources/templates/summarized_email_report.mustache]: typo in "range"

File: reporting/src/main/resources/templates/summarized_email_report.mustache [hunk 2]
```diff
@@ -155,13 +155,32 @@
...
+                </tr>
+                <tr>
+                    <td>
+                        <pre>
+ <b>Test Failures:</b> Number of builds(build consist of one infra combination) failed
+                       due to test failures.
+ <b>Infrastructure Errors:</b> Number of builds failed due to infrastructure failures.
+                          (e.g: Product deployment failures etc.)
+ <b>Tests Passed: </b>Tests Passed infra combination.
+                        </pre>
```
Comment (azinneera) [inline reporting/src/main/resources/templates/summarized_email_report.mustache]: Number of successful builds (build consist of one infra combination) with no test failures.

**Comment by yasassri** [comment]
> @yasassri can you post a sample screenshot on how this looks like?

@kasunbg Updated the description.

**Comment by kasunbg** [comment]
How about this: https://jsfiddle.net/t7ezpL30/430/

**Comment by yasassri** [comment]
> How about this: https://jsfiddle.net/t7ezpL30/430/

Updated the PR to have the description as a hover message.

**Pull request merged**